pwa-shell/src/ui/content-list/ContentList.js

```diff
@@ -27,33 +27,30 @@ class ContentList extends PureComponent {
   }
 
   componentDidMount() {
-    const { fetchContentList, isSearchResult, searchTerms } = this.props;
-    const filters = { searchTerms: isSearchResult ? searchTerms : '' };
-    fetchContentList(filters);
+    this.requestListWithFilter();
   }
 
   componentDidUpdate(prevProps) {
     if (this.props.contentType !== prevProps.contentType) {
-      const { fetchContentList, isSearchResult, searchTerms } = this.props;
-      const filters = { searchTerms: isSearchResult ? searchTerms : '' };
-      fetchContentList(filters);
+      this.requestListWithFilter();
     }
   }
 
+  requestListWithFilter(params = {}) {
```

Comment:
If we add filters to `fetchContentList`, could be easier to understand if we call it `fetchContentListWithFilter` then :-)

Reply:
done :)